pkg/config/config.go

```diff
@@ -54,12 +65,45 @@ type (
 )
 
 // FindByName find a table by its name.
-func (t Tables) FindByName(name string) (*Table, error) {
+func (t Tables) FindByName(name string) *Table {
 	for _, table := range t {
 		if table.Name == name {
-			return table, nil
+			return table
 		}
 	}
 
-	return nil, errors.New("table not found")
+	return nil
+}
+
+// LoadSpecFromFile loads klepto spec from file
+func LoadSpecFromFile(configPath string) (*Spec, error) {
+	if configPath != "" {
+		// Use config file from the flag.
+		log.Debugf("Reading config from %s ...", configPath)
+		viper.SetConfigFile(configPath)
+	} else {
+		log.Debugf("Reading config from %s ...", DefaultConfigFileName)
+
+		cwd, err := os.Getwd()
+		if err != nil {
+			return nil, wErrors.Wrap(err, "can't find current working directory")
+		}
```

Comment:
Should we put this in a `func init()` and set `DefaultConfigFileName` correctly from the get go?

Reply:
This part will go away completely. Actually let me just remove it here.

Reply:
removed